apps/expo/src/app/account.tsx

```diff
@@ -69,10 +74,48 @@ const NotificationContainer: React.FC<NotificationContainerProps> = ({
 const Account = () => {
   const { isSignedIn, user } = useUser();
 
+  const currentNotificationSettings =
+    api.user.getNotificationSettings.useQuery().data;
+  const updateNotifications = api.user.updateNotificationSettings.useMutation();
+
+  const [notifications, setNotifications] = useState({
+    [NotificationType.ANNOUCEMENTS]:
+      currentNotificationSettings?.notificationOnAnnoucements ?? false,
+    [NotificationType.COMMENTS_ON_POSTS]:
+      currentNotificationSettings?.notificationOnPostComments ?? false,
+    [NotificationType.LIKE_ON_POSTS]:
+      currentNotificationSettings?.notificationOnPostLikes ?? false,
+    [NotificationType.SCHEDULE_UPDATES]:
+      currentNotificationSettings?.notificationOnScheduleUpdates ?? false,
+  });
```

Comment:
is there a reason for this to be stored in local state if they are already in the notification settings hook, and are updated after a mutation?